src/Model/BaseBlock.php

```diff
@@ -202,10 +202,6 @@ public function hasParent(): bool
      */
     public function getTtl(): int
     {
-        if (false === $this->getSetting('use_cache', true)) {
-            return 0;
-        }
-
```

Comment:
Based on the comment, should this method be completely removed? 🤔

Reply:
Yes 👍 I will continue cleaning up in a follow up PR. Did not want to create one huge PR. Or would you prefer I do it all in one?

Reply:
yeah It's fine, didn't realise that the `ttl` property is not removed either 👍